contrib/agent-local.river

```diff
@@ -56,6 +56,10 @@ otelcol.exporter.otlp "default" {
   client {
     // TODO: Replace this with the endpoint for your trace receiver
     endpoint = env("TRACES_ENDPOINT")
+    tls {
```

Comment:
I had to add this. cc @theSuess

Reply:
Seems good. Haven't tested this with a local tempo setup, so thanks for checking this out!